Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license()" for more information.
>>> print(4)
4
>>> type ('Hi!')
<class 'str'>
>>> type(17)
<class 'int'>
>>> type(3.2)
<class 'float'>
>>> type('17')
<class 'str'>
>>> print(1000000)
1000000
>>> print(100,000,000)
100 0 0
>>> message = 'Okey, pognali?'
>>> n = 17
>>> pi = 3.1415926535897931
>>> print(n)
17
>>> print(pi)
3.141592653589793
>>> type(message)
<class 'str'>
>>> type(pi)
<class 'float'>
>>> type(n)
<class 'int'>
>>> 76trombones = <error, jo nosaukums nevar sakties no cipariem>
SyntaxError: invalid syntax
>>> more@ == <error, jo nosaukuma nevar but simbolie, ka @>
SyntaxError: invalid syntax
>>> class = <error, nevar pievienot mainigo klassam, vel ari tas ir aiznemts python keywords>
SyntaxError: invalid syntax
>>> str aiznemti = 'and del from None True tas elif global nonlocal try assert else if not while break except import or with class False in pass yield continue finally is raise async def for lambda return await'
SyntaxError: invalid syntax
>>> aiznemti = 'and del from None True tas elif global nonlocal try assert else if not while break except import or with class False in pass yield continue finally is raise async def for lambda return await'
>>> ls
Traceback (most recent call last):
  File "<pyshell#20>", line 1, in <module>
    ls
NameError: name 'ls' is not defined
>>> tikls = 'https://www.py4e.com/html3/02-variables kur paniemu info'
